src/main/java/teammates/ui/controller/AdminActivityLogPageData.java

```diff
@@ -276,15 +283,30 @@ private QueryParameters parseQuery(String query) throws Exception{
                     versions.add(values[j].replace(".", "-"));
                 }
                 
+            } else if (label.equals("from")) {
+                SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yy HH:mm");
+                Date d = sdf.parse(values[0] + " 00:00");                
+                Calendar cal = Calendar.getInstance();
+                cal.setTime(d);
+                cal = TimeHelper.convertToUserTimeZone(cal, - Const.SystemParams.ADMIN_TIMZE_ZONE_DOUBLE);
+                fromDateValue = cal.getTime().getTime();
+                
+            } else if (label.equals("to")) {
+                SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yy HH:mm");
+                Date d = sdf.parse(values[0] + " 23:59");                
+                Calendar cal = Calendar.getInstance();
```

Comment:
Shouldn't you use `TimeHelper.now(Const.SystemParams.ADMIN_TIMZE_ZONE_DOUBLE)` here to make sure it get the correct time in Singapore? I think contributors in different time zone may get different value of the calendar.